tests/phpunit/payment-tokens/class-wc-stripe-payment-tokens-test.php

```
@@ -267,6 +267,154 @@ public function provide_test_get_duplicate_token() {
+		$token->save();
+		$token_id = $token->get_id();
+
+		// New PM with same fingerprint but updated expiry.
+		$payment_method = $this->make_cc_payment_method( 'Fxxxxxxxxxxxxxxx', '2', '2031' );
+
+		$this->call_add_token_to_user( $payment_method, $this->make_mock_customer(), [ 'pm_old' ] );
+
+		$updated_token = WC_Payment_Tokens::get( $token_id );
+		$this->assertSame( '02', $updated_token->get_expiry_month() );
+		$this->assertSame( '2031', $updated_token->get_expiry_year() );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_token_to_user_updates_cc_expiry_on_duplicate is not named in camelCase. (undefined)

(CamelCaseMethodName)